internal/pkg/deploy/cloudformation/pipeline_integration_test.go

```diff
@@ -196,13 +203,43 @@ func TestPipelineCreation(t *testing.T) {
 			},
 			ArtifactBuckets: artifactBuckets,
 		}
-		require.NoError(t, projectDeployer.DeployPipeline(pipelineInput))
+		require.NoError(t, projectDeployer.CreatePipeline(pipelineInput))
```

Comment:
Does `DeployPipeline` simply not exist any more, or is there more behind this change?

Reply:
I guess DeployPipeline was split up into "Create" and "Update"